gencode.sh

```diff
@@ -20,4 +23,4 @@ do
   cat $entry >> $OUTPUT_FILE
 done
 
-Sourcery-0.10.1/bin/sourcery --verbose
+sourcery --verbose
```

Comment:
people will need to have sourcery in their paths but I think that this is okay 👍